src/System.Private.ServiceModel/tools/SelfHostedWcfService/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("SelfHostDriver")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("SelfHostDriver")]
+[assembly: AssemblyCopyright("Copyright ©  2016")]
```

Comment:
From what i see in CoreFx, there's no Copyright attribute in AssemblyInfo.cs, but the standard copyright notice appears at the top of AssemblyInfo.cs